spec/routes_spec.rb

```diff
@@ -9,14 +9,49 @@
   obj[klass] << action
 end
 
+pending_routes = YAML.safe_load(File.open(ManageIQ::UI::Classic.root.join('spec', 'config', 'routes.pending.yml')))
```

Comment:
This open the file handle but never closes it...Perhaps:

```suggestion
pending_routes = YAML.safe_load(ManageIQ::UI::Classic.root.join('spec', 'config', 'routes.pending.yml').read)
```